doc/Customization-guide/How-to-contribute/index.md

```diff
@@ -18,15 +18,15 @@ We would love to see you mastering the Kaa source code; however, writing code is
 Feel free to use all of the contribution options:
 
 - Join [Kaa Stack Overflow](http://stackoverflow.com/questions/tagged/kaa) to find answers to common user questions, ask your own Kaa-related questions, and help others in the community.
-- Use [Jira](http://jira.kaaproject.org/browse/KAA/) to provide feedback, make features and functionality suggestions, and help other community members.
+- Use [Jira](http://jira.kaaproject.org/) to provide feedback, make features and functionality suggestions, and help other community members.
```

Comment:
Add a link to Jira flow here, too. (As it is not limited to contributing to the source code.)